src/discord_utils.py

```diff
@@ -31,6 +31,26 @@ async def try_set_pin(message: Message, pin_state: bool) -> None:
         print("Altering message pin state failed:", e)
 
 
+def attachment_local_filepath(message_id: int, attachment: Attachment) -> str:
+    # Computed local filepath
```

Comment:
this comment doesn't really explain anything, can you add a short comment explaining why it's needed and what it does (assign unique local filename to remote discord attachment etc)